data/saas/config/datadog_config.yml

```diff
@@ -0,0 +1,59 @@
+saas_config:
+  - fides_key: datadog_connector_example
+    name: Datadog SaaS Config
+    type: datadog
+    description: A sample schema representing the Datadog connector for Fidesops
+    version: 0.0.1
+
+    connector_params:
+      - name: domain
+      - name: api_key
+      - name: app_key
+      - name: page_limit
+
+    client_config:
+      protocol: https
+      host: <domain>
+
+    test_request:
+      method: GET
+      path: /api/v2/logs/events
```

Comment:
Is there a way we can limit this to only return one event, or a smaller subset of events?

Reply:
no. It always returns a single type of data structure. There is another API but it aggregates the results and doesn't solve our problem.